cmd/commands/rook.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/rook"
+	"github.com/spf13/cobra"
+)
+
+// RookCmd represents the rook commands
+var RookCmd = &cobra.Command{
+	Use:   "rook",
+	Short: "Calls subcommands like `version`, `purge-osd, status` and etc.",
+	Args:  cobra.ExactArgs(1),
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Prints rook version",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		context := GetContext()
+		args = append(args, "version")
+		fmt.Println(k8sutil.RunCommandInOperatorPod(context, "rook", args, OperatorNamespace, CephClusterNamespace))
+	},
+}
+
+var purgeCmd = &cobra.Command{
+	Use:   "purge-osd",
+	Short: "Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		context := GetContext()
+		flag := cmd.Flag("force").Value.String()
```

Comment:
Does this flag works? This is used differently than I have seen flags used elsewhere in cobra, but maybe it's just a different pattern. In that case, maybe just a more specific name.
```suggestion
		forceFlag := cmd.Flag("force").Value.String()
```

Reply:
done